examples/pip_requirements/pip_requirements.py

```diff
@@ -46,23 +46,19 @@ def main():
     with mlflow.start_run() as run:
         run_id = run.info.run_id
 
-        # Get the expected mlflow version
-        mlflow_version_raw = Version(mlflow.__version__)
-        mlflow_version = f"mlflow=={mlflow_version_raw.major}.{mlflow_version_raw.minor}"
-
         # Default (both `pip_requirements` and `extra_pip_requirements` are unspecified)
         artifact_path = "default"
         mlflow.xgboost.log_model(model, artifact_path, signature=signature)
         pip_reqs = get_pip_requirements(run_id, artifact_path)
-        assert pip_reqs.issuperset([mlflow_version, xgb_req]), pip_reqs
```

Comment:
Removed `mlflow_version`. It's annoying if we make a change similar to #8687 in the future and this examples breaks again.